<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6KOFRV4" failures="3" tests="27" name="WebApp.BillGenrator.BillGen2023" time="396.653" errors="0" timestamp="2023-02-22T20:05:44 IST" skipped="1">
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFederalWithGrossPayYTDTotal40000" time="4.499">
    <failure message="The following asserts failed:
	expected [323.17] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [323.17] but found []
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at WebApp.BillGenrator.BillGen2023.verifyFederalWithGrossPayYTDTotal40000(BillGen2023.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFederalWithGrossPayYTDTotal40000 -->
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFICAMedicareWithGrossPayYTDTotalLessThan20000" time="5.059"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFICAMedicareWithGrossPayYTDTotalLessThan200000" time="5.317">
    <failure message="The following asserts failed:
	expected [117.45] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [117.45] but found []
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at WebApp.BillGenrator.BillGen2023.verifyFICAMedicareWithGrossPayYTDTotalLessThan200000(BillGen2023.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyFICAMedicareWithGrossPayYTDTotalLessThan200000 -->
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFICAMedicareWithGrossPayYTDTotalGreaterThan200000" time="4.532"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFICASocialSecurityCalculations" time="4.300"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFicaSocialSecurityCalculations" time="6.009"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyFederalTaxCalculationsonTemplate1" time="7.231"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyFederalTaxCalculationsWithMarriedOnTemplate1" time="7.442">
    <failure message="The following asserts failed:
	expected [22,905.42] but found [22,905.40],
	expected [22,905.42] but found [22,905.40]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [22,905.42] but found [22,905.40],
expected [22,905.42] but found [22,905.40]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at WebApp.BillGenrator.BillGen2023.VerifyFederalTaxCalculationsWithMarriedOnTemplate1(BillGen2023.java:163)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyFederalTaxCalculationsWithMarriedOnTemplate1 -->
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="verifyFederalTaxWithAllExemptions" time="21.311"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyStateTaxCalculationsWithAllStatesOnTemplate1" time="43.177"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyStateTaxCalculationsWithAllStatesWithSingleOnTemplate1" time="43.948"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyStateTaxCalculationsWithAllStatesWithSingle2OnTemplate1" time="43.496"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifyStateTaxCalculationsWithAllStatesWithMarriedOnTemplate1" time="43.291"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifySDITaxCalculationsonTemplate1" time="8.282"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifySDITaxCalculationsonTemplate1" time="8.292"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifySDITaxCalculationsonTemplate1" time="8.165"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifySDITaxCalculationsonTemplate1" time="8.220"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="VerifySDITaxCalculationsonTemplate1" time="8.356"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="Verify_SUI_Tax_Calculations_on_Template1_with_State" time="9.439"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="Verify_SUI_Tax_Calculations_on_Template1_with_State" time="9.409"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="Verify_SUI_Tax_Calculations_on_Template1_with_State" time="9.259"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="Verify_WC_Tax_Calculations_on_Template1_with_State" time="61.910"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="Verify_FLIFLV_Tax_Calculations_on_Template1_with_State" time="9.661"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="calculateWorkforceDevelopment" time="6.061"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="calculateWorkforceDevelopment2" time="4.931"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="calculateWorkforceDevelopment3" time="5.056"/>
  <system-out/>
  <testcase classname="WebApp.BillGenrator.BillGen2023" name="getexcelData">
    <skipped/>
  </testcase> <!-- getexcelData -->
  <system-out/>
</testsuite> <!-- WebApp.BillGenrator.BillGen2023 -->
